### DISCUSSION TITLE
How to run JUnit 5 Test Suites using this plugin

### DISCUSSION THREAD

**Discussion by nagkumar** [qualersoft/jmeter-gradle-plugin#174] [Q&A]
```
jmeter {
    // Configuration for JMeter
    testFiles = listOf("src/test/java/com/example/MyJUnit5TestSuite.kt") // Specify the path to your JUnit 5 Test Suite class

    testPlan {
	threads {
	    threadGroup(name = "JUnit Test Thread Group", threads = 10, rampUp = 5, loop = 2) {
		junitSampler(name = "JUnit 5 Sampler", className = "com.example.MyJUnit5TestSuite")
	    }
	}
    }
}
```

I really like such syntax as it is too clear to configure and run the load tests using JMeter and leveraging JUnit Tests.. This syntax is given by chatgpt, however, this script is not working...with this plugin

<img width="806" alt="image" src="https://github.com/qualersoft/jmeter-gradle-plugin/assets/332234/6938b452-a878-4927-9862-e43d0787c7ee">

Can any confirm the exact script to achieve the same need..

**Comment by nagkumar** [comment]
can you share insights on how to make this work.. pl..

**Comment by mathze** [comment]
:blush: Yeah ChatGPT isn't always your friend, I learned that a few times too :wink:.
The proposed syntax in not supported by this plugin, maybe another one.
But coming back to your question, this isn't possible with this plugin, as it is a simple replacement for JMeters CLI with a little sugar on top to manage the jmeter-tool and its dependencies.
